src/guide/components/props.md

```diff
@@ -484,7 +484,7 @@ Additional details:
 
 - An absent optional prop other than `Boolean` will have `undefined` value.
   
-- The `Boolean` absent props will be cast to `false`. You should set a `default` value for it in order to get desired behavior.
+- The `Boolean` absent props will be cast to `false`; a present `Boolean` prop in a template without value specified will be cast to `true` (see below). To control this behavior, you can instead set a `default` value.
```

Comment:
**question**: could you please provide an example of `present prop without a value specified`? The description is vague and I struggle to understand the case you are describing